test/common/pool.h

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2023 Intel Corporation
+// Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef UMF_TEST_POOL_H
+#define UMF_TEST_POOL_H
+
+#include <umf/memory_pool.h>
+
+#if defined(__cplusplus)
+extern "C" {
+#endif
+
+umf_memory_pool_handle_t nullPoolCreate(void);
```

Comment:
I think instead of having this file we should have two separate ones: one for null and one for tracing pool. Also, to create a pool we should use a standard API (umfPoolCreate) with OPS as an argument.

Reply:
Done.